### PR TITLE
Certificate, DNS, SMTP , Dataplane resource and datasource related ch…

### PR DESCRIPTION
**Pull request by shreyas3-vmw**
…anges

### CODE REVIEW

File: client/tdh/controller/service.go [hunk 1]
```diff
@@ -178,3 +178,40 @@ func (s *Service) GetClusterMetaData(id string) (*model.ClusterMetaData, error)
 
 	return &response, err
 }
+
+func (s *Service) GetCountercultureService() ([]model.ClusterCountByService, error) {
+	var response []model.ClusterCountByService
+
+	reqUrl := fmt.Sprintf("%s/%s/%s/%s", s.Endpoint, FleetManagement, SRE_cluster, Count)
```
Comment (sharmata-vmw) [inline client/tdh/controller/service.go:185]: what is the value of this `Count`?

Comment (sharmata-vmw) [reply client/tdh/controller/service.go:185]: ok, it is a path

File: client/tdh/infra-connector/query_certificate.go [hunk 2]
```diff
@@ -3,6 +3,7 @@ package infra_connector
 import "github.com/svc-bot-mds/terraform-provider-tdh/client/model"
 
 type CertificateQuery struct {
-	Name string `json:"name,omitempty"`
+	Name     string `json:"name,omitempty"`
```
Comment (sharmata-vmw) [inline client/tdh/infra-connector/query_certificate.go:6]: i think this also should be `schema` for conversion to query params

File: client/tdh/controller/service.go (hunk 1 above)
Comment (shreyas3-vmw) [reply client/tdh/controller/service.go:185]: yes. Api Url const

**Review by sharmata-vmw — APPROVED** [review]

**Pull request merged**